tests/trainer/logging_/test_distributed_logging.py

```diff
@@ -45,12 +72,14 @@ def test_global_zero_only_logging_ddp_cpu(tmpdir):
         limit_val_batches=1,
         max_epochs=1,
         weights_summary=None,
+        logger=all_rank_logger,
+        log_every_n_steps=1,
     )
     trainer.fit(model)
 
 
 @RunIf(min_gpus=2)
-def test_global_zero_only_logging_ddp_spawn(tmpdir):
+def test_all_rank_logging_ddp_spawn(tmpdir):
```

Comment:
@edward-io  this test is failing. by default, the trainer uses a tensorboard logger, which defines its `experiment` only on rank 0. so the assertion on L57 in the TestModel is failing because `self.logger.logs` is not populated with that logger.